.eslintignore

```diff
@@ -3,4 +3,6 @@ third-party
 **/*.json
 src/shared/telemetry/clienttelemetry.d.ts
 **/*.gen.ts
+plugins/**
 src/testFixtures/workspaceFolder/ts-plain-sam-app/
+.eslintrc.js
```

Comment:
why not lint `plugins/` or this file?

Reply:
I disabled plugins temporarily but guess I forgot to add it back.

For `.eslintrc.js` there's a really weird linter error that simply does not go away even on mainline. Will add it back when I can figure out what it is.